README.md

```diff
@@ -205,24 +205,40 @@ Things you need:
     parameters send to that URL)
 - A Fortnox test environment so that you can test your integration.
 
-When you have authorized your integration you get an access token. It's a JWT
-with an expiration time of 1 hour. You also get a refresh token that lasts for
-**31 days**. When a new access token is requested, a new refresh token is also
-provided and the old one is invalidated. As long as the refresh token is valid,
-the **gem will do all of this automatically**. **You just need to make sure the
-gem makes a request the Fortnox API before the current refresh token expires**,
-otherwise you need to start over again with the
-[Initialization](#initialization).
+When you have authorized your integration you get an access token from Fortnox.
+It's a JWT with a expiration time (currently **1 hour**). You also get a long
+lived refresh token (currently lasts for **31 days** ). When you need a new
+access token you send a renewal request to Fortnox. That request contains the
+new access token as well as a new refresh token and some other data. Note that
+**the old refresh token is invalidated when new tokens are requested**. As long
+as you have a valid refresh token you will be available to request new tokens.
+
+The gem exposes a specific repository for renewing tokens. You use it like this:
+
+```ruby
+require 'fortnox/api'
 
-Note that when the gem automatially fetches a new access token, both the client
-id and client secret are required to generate the authorization headers for that
-request. Therefore, the gem needs access to those values as well.
+# Request new tokens
+tokens = Fortnox::API::Repository::Authentication.new.renew_tokens(your_refresh_token)
 
-## Initialization
+# You probably want to persist your tokens somehow...
+store_refresh_token(tokens[:refresh_token])
+store_access_token(tokens[:access_token])
+
+# Set the new access token
+Fortnox::API.access_token = tokens[:access_token]
+
+# The gem will now use the new token
+Fortnox::API::Repository::Customer.new.all
+```
+
+It's up to you to provide a valid token to the gem and to renew it regularly,
+otherwise you need to start over again with the
+[Initialization](#initialization).
 
-As described in the [Authorization](#authorization) section, the gem refreshes
-the tokens automatically, but you need to initialize the process. There a script
-for getting tokens. See [Get tokens](#get-tokens).
+Note that when the gem fetches new tokens, both the client id and client secret
```

Comment:
"when the gem fetches" sounds like it is doing it automatically, but we are not doing that anymore, right? So maybe "when you use the gem to fetch" instead? Making it a passive sentence (from the gems pov)